partials/form.html

```diff
@@ -0,0 +1,54 @@
+<h3>{{title}}</h3>
+<form>
+    <div class="form-group">
+        <label>Horse Haven ID</label>
+        <input type="text" class="form-control" placeholder="Unique Id" ng-model="horse.horseHavenId"/>
+    </div>
+    <div class="form-group">
+        <label>Name</label>
+        <input type="text" class="form-control" placeholder="Horse's Name" ng-model="horse.name"/>
+    </div>
+    <div class="form-group">
+        <label>Breed</label>
+        <input type="text" class="form-control" placeholder="Breed" ng-model="horse.breed"/>
+    </div>
+    <div class="form-group">
+        <label>Color</label>
+        <input type="text" class="form-control" placeholder="Color" ng-model="horse.color"/>
+    </div>
+    <div class="form-group">
+        <label>Gender</label>
+        <label >
+            <input type="radio" class="form-control" name="breed" ng-model="horse.gender.stallion">
+            Stallion
+        </label>
+        <label >
+            <input type="radio" class="form-control" name="breed" ng-model="horse.gender.gelding">
+            Gelding
+        </label>
+        <label >
+            <input type="radio" class="form-control" name="breed" ng-model="horse.gender.mare">
+            Mare
+        </label>
+        <div class="form-group">
+            <label>Age</label>
+            <input type="text" class="form-control" placeholder="Age" ng-model="horse.age"/>
+        </div>
+        <div class="form-group">
+            <label>County Of Origin</label>
+            <input type="text" class="form-control" placeholder="County of Origin" ng-model="horse.countyOfOrigin"/>
+        </div>
+        <div class="form-group">
+            <label>Arrival Date</label>
+            <input type="date" class="form-control" placeholder="Arrival Date" ng-model="horse.arrivalDate"/>
```

Comment:
Try the datetime type for the nice graphical picker, at least I THINK that's it name...